mParticleCore.brs

```diff
@@ -926,12 +926,15 @@ function mParticleStart(options as object, messagePort as object)
                 localSeconds = tempDatetime.AsSeconds ()
                 utcSecondsOffset = localSeconds - utcSeconds
                 utcHoursOffset = utcSecondsOffset / 3600
+                versionArray = info.GetOsVersion()
+                versionString = Substitute("Version: {0}.{1} Revision: {2} Build: {3}", versionArray.major, versionArray.minor, versionArray.revision, versionArray.build)
```

Comment:
I think it'd be better just to have this be {0}.{1}.{2}.{3} - easier to parse for anything downstream